dali/base/dadfs.cpp

```diff
@@ -5792,15 +5792,23 @@ class CDistributedSuperFile: public CDistributedFileBase<IDistributedSuperFile>
     __int64 getRecordCount()
     {
         __int64 ret = queryAttributes().getPropInt64("@recordCount",-1);
-        if (ret==-1) {
+        if (ret==-1)
+        {
             ret = 0;
-            ForEachItemIn(i,subfiles) {
+            ForEachItemIn(i,subfiles)
+            {
+                IDistributedFile &subFile = subfiles.item(i);
                 __int64 rc = subfiles.item(i).queryAttributes().getPropInt64("@recordCount",-1);
```

Comment:
minor - why not use subFile here:
__int64 rc = subFile.queryAttributes().getPropInt64("@recordCount",-1);
or dont use subFile at all ?

Reply:
yep, should have changed that line to use subFile, will change.